community-membership.md

```diff
@@ -221,11 +221,39 @@ The following apply to the subproject for which one would be an owner.
 - Ensure a healthy process for discussion and decision making is in place.
 - Work with other subproject owners to maintain the project's overall health and success holistically
 
-## ~~Maintainer~~
+## Inactive members
 
-**Status:** Removed
+_Members are continuously active contributors in the community._
+
+A core principal in maintaining a healthy community is encouraging active
+participation. It is inevitable that people's focuses will change over time and
+they are not expected to be actively contributing forever.
+
+However, being a member of one of the Kubernetes GitHub organizations comes with
+an [elevated set of permissions]. These capabilities should not be used by those
+that are not familiar with the current state of the Kubernetes project.
+
+Therefor members with an extended period away from the project with no activity
+will be removed from the Kubernetes Github Organizations and will be required to
+go through the org membership process again after re-familiarizing themselves
+with the current state.
+
+
+### How inactivity is measured
+
+Inactive members are defined as members of one of the Kubernetes Organizations
+with **no** contributions across any organization within 18 months. This is
+measured by the CNCF [DevStats project].
+
+**Note:** Devstats does not take into account non-code contributions. If a
```

Comment:
I'm not sure this is strictly true, unless I'm misreading https://github.com/cncf/devstats/blob/master/metrics/shared/project_developer_stats.sql#L104-L120 I think participation in issues and reviews counts as contribution

Reply:
It does count quite a bit, issues, actions etc on PRs. This block is more so to cover things like slack admin duties, helping with events, moderating discuss.kubentes.io etc. Those things don't always have an issue to back them up.